<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBar</class>
 <widget class="QWidget" name="SearchBar">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小命令工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F0F0F0;
            color: #333333;
            font-size: 16px;
            font-family: 'Helvetica Neue', Arial, sans-serif;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF;
            border-radius: 20px;
            border: 1px solid #CCCCCC;
            padding: 10px;
            font-size: 18px;
            font-family: 'Helvetica Neue', Arial, sans-serif;</string>
         </property>
         <property name="placeholderText">
          <string>开始输入命令...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: #007BFF;
            border-radius: 20px;
            border: none;
            padding: 10px 20px;
            font-size: 18px;
            font-family: 'Helvetica Neue', Arial, sans-serif;
            color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF;
selection-background-color: #FFCA72;
border-radius: 10px;
border: 1px solid #CCCCCC;
padding: 10px;
font-size: 20px;
font-family: 'Helvetica Neue', Arial, sans-serif;
line-height: 1.5;
margin-bottom: 10px;	
margin: 1px; 
background: rgba(255, 255, 255, 0.9);</string>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>你好</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hello</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
